packages/react-examples/.storybook/preview.js

```diff
@@ -62,7 +64,9 @@ function loadStories() {
 
   for (const req of contexts) {
     req.keys().forEach(key => {
-      generateStoriesFromExamples(key, stories, req);
+      if (includeE2E || !key.includes('/e2e/')) {
```

Comment:
do stories in `react-examples` actually show up in the docsite ? I thought that it was an internal only dev loop